### DISCUSSION TITLE
bin field

### DISCUSSION THREAD

**Discussion by PRMSA** [mercadopago/sdk-js#73] [General]
O campo bin é para ter 6 ou 8 digitos? pois nos documentos do site e exemplos de código utilizam 6 digitos 
ex.: `let bin = cardnumber.substring(0,6);
 window.Mercadopago.getPaymentMethod({
           "bin": bin
       }, setPaymentMethod);` mas na documentação aqui do github diz que ele deve ter 8 dígitos ex.: 
Option Key | Type | Description |  
-- | -- | -- | --
bin | STRING | Card number first 8 digits

então é pra ser 8 ou 6?

**Comment by lucmantovani** [comment]
Olá, @PRMSA!

A partir de Abril de 2022 passaram a ser emitidos cartões com BIN de tamanho 8 dígitos, então recomendamos utilizar sempre 8 dígitos.

Vou atualizar as ocorrências aqui na documentação técnica, obrigado por avisar!
Consegue me enviar em qual página do site ainda está apresentando BIN com 6 dígitos?

Obrigado!

**Reply by PRMSA** [reply]
em todas as páginas do site mercadopago que falam sobre a api, pelo menos todas as q vi. 
A api rest só retorna os primeiros 6 digitos dos cartões salvos não?

**Reply by lucmantovani** [reply]
Consegue me enviar o link de uma página, por favor?

Sim, por enquanto a API de cartões salvos somente retorna os 6 primeiros dígitos.
Mas nesse caso, por já ter o cartão armazenado, o BIN não é necessário. Basta utilizar o `cardId` e solicitar novamente o `CVV`

**Reply by PRMSA** [reply]
acho que só vi essa aqui https://www.mercadopago.com.br/developers/en/docs/checkout-api-v1/receiving-payment-by-card
os outros se encaixa no que vc explicou.

essa área pode fazer perguntas sobre outras partes da api ou só do mercadopago.js?

**Reply by lucmantovani** [reply]
Realmente nesse guia está desatualizado, mas provavelmente será atualizado em breve.
Somente um detalhe: o guia utiliza a versão antiga da SDK JS, que já não recebe mais suporte ativo. Recomendamos atualizar para a nova versão, descrita nesse [guia](https://www.mercadopago.com.br/developers/pt/docs/checkout-api/payment-methods/receiving-payment-by-card)
A versão 2 conta com algumas funcionalidades a mais, e também melhorias de segurança 🙂

Aqui concentramos dúvidas/relatos específicos das SDKs
Outras dúvidas relacionadas à integração devem ser direcionadas ao [suporte](https://www.mercadopago.com.br/developers/pt/support/center/tickets)

**Reply by PRMSA** [reply]
puts tinha umas perguntas sobre a integração agora só no teste e erro.
